src/index.js

```diff
@@ -18,7 +18,17 @@ function run(argv, env) {
   const rl = readline.createInterface({ input: cp.stdout });
 
   cp.on("error", (err) => {
-    console.error(err);
+    if (options.cmd === 'cargo' && err.code === 'ENOENT') {
+      console.error(`Error: could not find the \`cargo\` executable.
+
+If you don't have Rust on your system, you can install it by running:
+
+    curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
```

Comment:
I think we should link to the Rust lang project page instead of providing specific install commands:
* The project might change the recommended install method and we would need to update
* The install method changes per platform (this doesn't work on Windows)

Reply:
Great points! How would you feel about linking directly to the Rust installation page, as a middle ground? That URL could change but it probably won't change often. And every step we eliminate prevents an additional % of potential new users from giving up.

Reply:
That sounds perfect!